tests/tests_twobody/test_orbit.py

```diff
@@ -571,27 +560,6 @@ def test_from_horizons_raise_valueerror():
     )
 
 
-@pytest.mark.remote_data
-def test_orbit_from_horizons_has_expected_elements():
-    epoch = Time("2018-07-23", scale="tdb")
-    # Orbit Parameters of Ceres
-    # Taken from https://ssd.jpl.nasa.gov/horizons.cgi
-    ss = Orbit.from_classical(
-        Sun,
-        2.76710759221651 * u.au,
-        0.07554803091400027 * u.one,
-        27.18502494739172 * u.deg,
-        23.36913218336299 * u.deg,
-        132.2919809219236 * u.deg,
-        21.28957916690369 * u.deg,
-        epoch,
-    )
-    ss1 = Orbit.from_horizons(name="Ceres", attractor=Sun, epoch=epoch)
```

Comment:
Time to remove `Orbit.from_horizons` :)

Reply:
Just to confirm what you meant: We should remove `Orbit.from_horizons` from `src` and their corresponding tests, right?

Sorry, I was confused with our intention regarding `Orbit.from_horizons`..

Reply:
Yes, let's remove it! It was deprecated in 0.14 I think